letsencrypt_transip_nl/models/letsencrypt.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models
+from tempfile import NamedTemporaryFile
+import base64
+import logging
+_logger = logging.getLogger(__name__)
+
+try:
+    from transip.service.objects import DnsEntry
+    from transip.service import DomainService
+except ImportError as err:
+    _logger.debug(err)
+
+
+class Letsencrypt(models.AbstractModel):
+    _inherit = 'letsencrypt'
+
+    def letsencrypt_transip_support(self, challenge, domain):
```

Comment:
you'll have to rename this function depending on what you do with https://github.com/OCA/server-tools/pull/1194/files#r230787313